### ISSUE TITLE
Does not work with PHP 7.2

### ISSUE THREAD

**Issue by sebastianbergmann** [localheinz/github-pulse#41]
```
PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /home/sb/github-pulse/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php on line 67
PHP Stack trace:
PHP   1. {main}() /home/sb/github-pulse/github-pulse:0
PHP   2. include() /home/sb/github-pulse/github-pulse:4
PHP   3. Symfony\Component\Console\Application->run() /home/sb/github-pulse/github-pulse.php:76
PHP   4. Symfony\Component\Console\Application->doRun() /home/sb/github-pulse/vendor/symfony/console/Application.php:124
PHP   5. Symfony\Component\Console\Application->doRunCommand() /home/sb/github-pulse/vendor/symfony/console/Application.php:200
PHP   6. Localheinz\GitHub\Pulse\Console\GenerateCommand->run() /home/sb/github-pulse/vendor/symfony/console/Application.php:835
PHP   7. Localheinz\GitHub\Pulse\Console\GenerateCommand->execute() /home/sb/github-pulse/vendor/symfony/console/Command/Command.php:264
PHP   8. Localheinz\GitHub\Pulse\Repository\OrganizationRepository->find() /home/sb/github-pulse/src/Console/GenerateCommand.php:149
PHP   9. Localheinz\GitHub\Pulse\Repository\OrganizationRepository->resourceFromApi() /home/sb/github-pulse/src/Repository/OrganizationRepository.php:27
PHP  10. Github\Api\Organization->show() /home/sb/github-pulse/src/Repository/OrganizationRepository.php:53
PHP  11. Github\Api\Organization->get() /home/sb/github-pulse/vendor/knplabs/github-api/lib/Github/Api/Organization.php:39
PHP  12. Http\Client\Common\HttpMethodsClient->get() /home/sb/github-pulse/vendor/knplabs/github-api/lib/Github/Api/AbstractApi.php:81
PHP  13. Http\Client\Common\HttpMethodsClient->send() /home/sb/github-pulse/vendor/php-http/client-common/src/HttpMethodsClient.php:61
PHP  14. Http\Client\Common\HttpMethodsClient->sendRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/HttpMethodsClient.php:192
PHP  15. Http\Client\Common\PluginClient->sendRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/HttpMethodsClient.php:203
PHP  16. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:87
PHP  17. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:170
PHP  18. Github\HttpClient\Plugin\GithubExceptionThrower->handleRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:156
PHP  19. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/knplabs/github-api/lib/Github/HttpClient/Plugin/GithubExceptionThrower.php:26
PHP  20. Http\Client\Common\Plugin\HistoryPlugin->handleRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:156
PHP  21. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/client-common/src/Plugin/HistoryPlugin.php:39
PHP  22. Http\Client\Common\Plugin\RedirectPlugin->handleRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:156
PHP  23. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/client-common/src/Plugin/RedirectPlugin.php:145
PHP  24. Http\Client\Common\Plugin\AddHostPlugin->handleRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:156
PHP  25. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/client-common/src/Plugin/AddHostPlugin.php:64
PHP  26. Http\Client\Common\Plugin\HeaderDefaultsPlugin->handleRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:156
PHP  27. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/client-common/src/Plugin/HeaderDefaultsPlugin.php:41
PHP  28. Http\Client\Common\Plugin\HeaderAppendPlugin->handleRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:156
PHP  29. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/client-common/src/Plugin/HeaderAppendPlugin.php:43
PHP  30. Http\Client\Common\Plugin\CachePlugin->handleRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:156
PHP  31. Http\Client\Common\PluginClient->Http\Client\Common\{closure}() /home/sb/github-pulse/vendor/php-http/cache-plugin/src/CachePlugin.php:157
PHP  32. Http\Adapter\Guzzle6\Client->sendRequest() /home/sb/github-pulse/vendor/php-http/client-common/src/PluginClient.php:81
PHP  33. Http\Adapter\Guzzle6\Promise->wait() /home/sb/github-pulse/vendor/php-http/guzzle6-adapter/src/Client.php:56
PHP  34. GuzzleHttp\Promise\Promise->wait() /home/sb/github-pulse/vendor/php-http/guzzle6-adapter/src/Promise.php:95
PHP  35. GuzzleHttp\Promise\Promise->waitIfPending() /home/sb/github-pulse/vendor/guzzlehttp/promises/src/Promise.php:62
PHP  36. GuzzleHttp\Promise\Promise->invokeWaitList() /home/sb/github-pulse/vendor/guzzlehttp/promises/src/Promise.php:225
PHP  37. GuzzleHttp\Promise\Promise->waitIfPending() /home/sb/github-pulse/vendor/guzzlehttp/promises/src/Promise.php:267
PHP  38. GuzzleHttp\Promise\Promise->invokeWaitFn() /home/sb/github-pulse/vendor/guzzlehttp/promises/src/Promise.php:223
PHP  39. GuzzleHttp\Handler\CurlMultiHandler->execute() /home/sb/github-pulse/vendor/guzzlehttp/promises/src/Promise.php:246
PHP  40. GuzzleHttp\Handler\CurlMultiHandler->tick() /home/sb/github-pulse/vendor/guzzlehttp/guzzle/src/Handler/CurlMultiHandler.php:123
PHP  41. GuzzleHttp\Handler\CurlMultiHandler->processMessages() /home/sb/github-pulse/vendor/guzzlehttp/guzzle/src/Handler/CurlMultiHandler.php:108
PHP  42. GuzzleHttp\Handler\CurlFactory::finish() /home/sb/github-pulse/vendor/guzzlehttp/guzzle/src/Handler/CurlMultiHandler.php:179
PHP  43. GuzzleHttp\Handler\CurlFactory->release() /home/sb/github-pulse/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php:107
PHP  44. count() /home/sb/github-pulse/vendor/guzzlehttp/guzzle/src/Handler/CurlFactory.php:67
```

**Comment by sebastianbergmann** [comment]
I do not know whether there is a newer version of Guzzle that supports PHP 7.2.

**Comment by localheinz** [comment]
@sebastianbergmann 

Thank you for your report! I will take a look as soon as I can. Unfortunately quite busy at the moment.

**Comment by shooding** [comment]
Same issue in guzzlehttp/guzzle": "6.2.*"
Solved in guzzlehttp/guzzle": "6.3.3"

**Comment by localheinz** [comment]
@sebastianbergmann @shooding 

Can you try with current `master`?

I ran

```
$ php -v

PHP 7.2.8 (cli) (built: Jul 19 2018 12:15:24) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.8, Copyright (c) 1999-2018, by Zend Technologies
```

and then

```
$ composer global require localheinz/github-pulse:dev-master#171187f
```

and then

```
$ github-pulse generate bounceio
```

which yielded (for me)

```
Localheinz GitHub Pulse
=======================

Going to determine the organization wide pulse:

 * in organization "bounceio"
 * between "1970-01-01 00:00:00" and "2018-09-14 12:03:44"

Repositories in "bounceio"
--------------------------

 * bounce
 * cartographer
 * emitter


 [OK] Found 3 repositories


Processing repositories in "bounceio"
-------------------------------------

 // This may take a while and fail when the GitHub API rate limit is exhausted. Run it again, then.

 * bounce (2 pull requests)
 * cartographer (0 pull requests)
 * emitter (0 pull requests)

Pulse based on commits of merged pull requests in organization "bounceio" between "1970-01-01 00:00:00" and "2018-09-14 12:03:44"
---------------------------------------------------------------------------------------------------------------------------------

 --- ------------ --------------------------------- ---------------------- ---------------------- ----------------------
  #   User         Commits of merged pull requests   Merged pull requests   First PR               Last PR
 --- ------------ --------------------------------- ---------------------- ---------------------- ----------------------
  1   shrikeh      15                                1                      2017-09-10T09:18:44Z   2017-09-10T09:18:44Z
  2   localheinz   1                                 1                      2017-09-10T09:36:26Z   2017-09-10T09:36:26Z
 --- ------------ --------------------------------- ---------------------- ---------------------- ----------------------


 [OK] Found 2 merged pull request(s) in organization "bounceio" between "1970-01-01 00:00:00" and "2018-09-14 12:03:44"


Time: 2 secs, Memory: 6.0 MiB.
```
